WMF Framework/Configuration.swift

```diff
@@ -182,17 +182,17 @@ public class Configuration: NSObject {
     }
     
     /// Returns the default request headers for Page Content Service API requests
-    public func pageContentServiceHeaders(for wikipediaLanguage: String? = nil) -> [String: String] {
-        guard let apiLanguage = wikipediaLanguage else {
-            return [:]
-        }
+    public func pageContentServiceHeaders(for url: URL) -> [String: String] {
```

Comment:
I don't think we need this method anymore now that `Session`'s default header attempts to assign a single variant at first. This was added at the time because the backend needed a single variant rather than `Session`'s exhaustive list, otherwise it would send a response with mixed characters.

Reply:
I should have mentioned that in the PR review. I have that on a list of things to look at after I get the bigger features in the app working. For now, I had wanted to leave it in, as one less behavior change among all the other changes. But I also think by the end of the feature this method will be gone, I just want to take the time to test and double check before removing it.